go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/pkg/sftp v1.11.0
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.45.0
+	github.com/spf13/cobra v1.2.1
```

Comment:
Can we use the a more recent version (i.e. [v1.4.0](https://github.com/spf13/cobra/releases/tag/v1.4.0))?

Reply:
Good point, updated